### PR TITLE
Renaming the first topping

### PR DESCRIPTION
**Pull request by KieraDOG**
Renaming the first topping from 'Anchovy' to 'I'.

### CODE REVIEW

**Comment by yhong4** [comment]
asfddafdsaf

**Comment by BubbleKID** [comment]
阅

**Comment by KieraDOG** [comment]
Account bussiness requirments, we need rename the first item

File: src/example.html [hunk 1]
```diff
@@ -197,7 +197,7 @@
 
     function renderToppingsForm() {
       var toppings = [{
-        name: 'Anchovy',
+        name: 'I',
```
Comment (KieraDOG) [inline src/example.html]: Is this implementation good?

**Pull request merged**